Blish HUD/_Utils/DirectoryUtil.cs

```diff
@@ -36,10 +36,22 @@ public static class DirectoryUtil {
 
         static DirectoryUtil() {
             // Prepare user documents directory
-            BasePath = ApplicationSettings.Instance.UserSettingsPath
-                    ?? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments,
-                                                               Environment.SpecialFolderOption.DoNotVerify), 
-                                    ADDON_DIR);
+            // Check if current working directory contains "Settings" folder
+            // in that case override MyDocuments location as default for portability
+            // --settings cli argument has still priority
+            string BlishExeFilePath = System.Reflection.Assembly.GetExecutingAssembly().Location;
```

Comment:
No need to pull it this way.  You can pull it by the current directory under Directory since we've set it here:

https://github.com/blish-hud/Blish-HUD/blob/fe56e84006b5aa3deae89c85faa4452fb1c727fc/Blish%20HUD/Program.cs#L43